.github/workflows/prodsec-workflow.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
     - main
     - develop
+    - cspl-2322
```

Comment:
Just a reminder to remove CSPL-2322 before merge.

Reply:
removed